src/SFA.DAS.ApplyService.Web/Views/Application/PagesWithSections.cshtml

```diff
@@ -72,17 +71,19 @@
 
                                 @if (nextPageTitle != pageTitle)
                                 {
-                                    @Html.Raw("</ul>");
+                                    @Html.Raw("</ul></li>");
```

Comment:
this </li> bit looks wrong to me

Reply:
I'm pretty sure that's right. I know what you're getting at, but it's closing a list item which contains an unordered list so it looks the wrong way round.